src/main/java/com/aws/greengrass/builtin/services/mqttproxy/MqttProxyIPCAgent.java

```diff
@@ -202,7 +204,20 @@ private void forwardToSubscriber(MqttMessage message) {
         }
     }
 
-    private String validateTopic(String topic, String serviceName) {
+    private String validatePublishTopic(String topic, String serviceName) {
```

Comment:
This is already checked in the MqttClient itself, so don't duplicate effort here.

Reply:
oh, thanks for pointing that out